PROTO_tests/tests/pdarray_creation_test.py

```diff
@@ -560,6 +567,9 @@ def test_random_strings_lognormal_errors(self):
         with pytest.raises(TypeError):
             ak.random_strings_lognormal(2, 0.25, 100, 1000000)
 
+        # Test that int_scalars covers uint8, uint16, uint32
+        ak.random_strings_lognormal(np.uint8(2), 0.25, np.uint16(100))
```

Comment:
I think if we instantiate an array we should check the output is reasonable.

Reply:
I agree, but as above, I'm copying what was in the soon-to-be-deprecated version.  I suspect this is just there to see if an error is thrown.